db/commitlog/commitlog.cc

```diff
@@ -110,6 +110,10 @@ db::commitlog::config db::commitlog::config::from_db_config(const db::config& cf
     c.use_o_dsync = cfg.commitlog_use_o_dsync();
     c.allow_going_over_size_limit = !cfg.commitlog_use_hard_size_limit();
 
+    if (cfg.commitlog_flush_threshold_in_mb() >= 0) {
+        c.commitlog_flush_threshold_in_mb = cfg.commitlog_flush_threshold_in_mb();
```

Comment:
If we make this a parameter, let's listen to it and make it liveupdate so we don't need to restart.

Reply:
Can we do this an a coming exercise? This is converted to a commitlog::config object, passed by value. Doing a changeable thing here will make for very ugly code (not the least because config values are per node, commitlog ones are per shard).

Reply:
updateable_value was supposed to address exactly that problem, unfortunately it has a cross-shard bug. Okay to defer it.

Reply:
Since the configs are separate, the best solution (methinks) would be to add observers in database, and simply re-apply the full  config on change (or rather the parts that can be changed). Still a little over-ugly for this patch set. 
Could also create two different initializers for CL, one with db::config, one with local (for hints).